src/proto/NULS.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+
+package TW.NULS.Proto;
+option java_package = "wallet.core.jni.proto";
+
+message TransactionCoinFrom {
```

Comment:
Are the naming `CoinFrom` and `CoinTo` also used in NULS code base? can we rename to Input/Output?

Reply:
They are used by NULS code only and their names are same with NULS definition.
I think they need not to be changed.